h/static/scripts/app.coffee

```diff
@@ -1,3 +1,11 @@
+angular = require('angular')
+uuid = require('./vendor/uuid')
+
+# These services are provided in their own angular modules and thus must be
```

Comment:
Never mind. Of course. You need to have the module defined here before you import other things that try to declare controllers, etc on it.